utils/cluster_manager.py

```diff
@@ -258,7 +260,7 @@ def create_cluster_folder(path: str, prefix: str) -> str:
         str: The full path of the cluster folder
     """
     time = datetime.now(timezone.utc)
-    time_str = time.strftime("%Y-%m-%dT%H:%M:%SZ")
+    time_str = time.strftime("%Y-%m-%dT%H-%M-%SZ")
```

Comment:
why?

Reply:
1. It is not compatible with Windows/WLS
2. [GHA blames and ignores such paths](https://github.com/Bit-Quill/glide-for-redis/actions/runs/7645825241/job/20833302458#step:7:22), so we can't have cluster logs without that fix